db/migrate/20230824130826_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
As per the ER diagram, the association between the user and like is set up through the AuthorId and not directly on the User. You can use the same approach as you did for the posts.
![image](https://github.com/uno36/blog-app-rails/assets/33268927/4d72a7cd-3833-449c-9bec-61eedddc690a)